toolkits/global/packages/global-card/HISTORY.md

```diff
@@ -1,5 +1,9 @@
 # History
 
+## 5.0.0 (2021-11-08)
```

Comment:
Just based on the breaking change below, should it be structured like this?

```
* BREAKING:
    * bla bla
```

I don't know if that's a _thing_ or just personal preferences. Just caught my eye.

Reply:
@tulipdexter You're right, it is like that normally.